### ISSUE TITLE
Missing HTML lang attribute

### ISSUE THREAD

**Issue by urre** [TypeStrong/typedoc#1951]
## Search terms

Missing `lang` attribute on the HTML tag. The "lang" attribute sets the page language. Screen readers rely on this information to determine which language library to use. The missing `lang` attribute results in doc pages getting issues in SEO audits.

## Expected Behavior

To be able to set `lang` from some config

**Comment by urre** [comment]
👏
